Competition2017/src/competition/subsystems/drive/commands/AutonomousDriveAlongCurveCommand.java

```diff
@@ -11,69 +11,79 @@
 import xbot.common.properties.DoubleProperty;
 import xbot.common.properties.XPropertyManager;
 
-public class DriveAlongCurveCommand extends BaseDriveCommand {
+public class AutonomousDriveAlongCurveCommand extends BaseDriveCommand {
```

Comment:
Can we call this `Path` instead of `Curve`? In reality, we're giving it points that are likely multiple curves.